dist/legislation-changes/index.html

```diff
@@ -80,12 +80,37 @@
         <h2>Keep Informed</h2>
         <ul>
           <li>
+<<<<<<< HEAD
```

Comment:
These things are not supposed to be here. These are remnants of the merge conflicts. Run `git mergetool` or look for <<<<<< and >>>>>> in your code.